airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -807,6 +807,10 @@ definitions:
         title: Whether the target API does not support filtering and returns all data (the cursor filters records in the client instead of the API side)
         description: If the target API endpoint does not take cursor values to filter records and returns all records anyway, the connector with this cursor will filter out records locally, and only emit new records from the last sync, hence incremental. This means that all records would be read from the API, but only new records will be emitted to the destination.
         type: boolean
+      is_compare_strictly:
+        title: Whether the target API should return an error for slice intersections
+        description: If the target API endpoint does not accept datetime filtering fields with the same value, the connector will perform slicing without intersections.
+        type: boolean
```

Comment:
let's add default: False for extra clarity

Reply:
updated